Meadowlark-js/packages/meadowlark-core/src/validation/DocumentIdentityExtractor.ts

```diff
@@ -127,6 +127,11 @@ export function extractDocumentIdentity(entity: TopLevelEntity, body: object): D
     },
     { documentIdentity: NoDocumentIdentity, studentId: null, edOrgId: null },
   );
+
+  // Ensure proper ordering of identity fields, by name value ascending
+  result.documentIdentity.sort((a, b) => a.name.localeCompare(b.name));
+
```

Comment:
Strictly speaking unnecessary today, but without it we are relying on the MetaEd stuff to always order it's internal references properly for us. It would be kind of a catastrophe if that changed and we started hashing bad ids.